jwt/jwt.go

```diff
@@ -0,0 +1,239 @@
+package jwt
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var (
+	jwtUtilsInstance *JWTUtils
+	initError        error
+	once             sync.Once
+)
+
+type JWTUtils struct {
+	publicKey *rsa.PublicKey
+}
+
+type EnvConfig struct {
+	SessionCookieName string
+	Environment       string
+}
+
+func LoadEnvConfig() (*EnvConfig, error) {
+	return &EnvConfig{
+		SessionCookieName: os.Getenv("SESSION_COOKIE_NAME"),
+		Environment:       os.Getenv("ENVIRONMENT"),
+	}, nil
+}
```

Comment:
- can we make this a package instead of keeping this inside this file ?
    - example: 
    1. https://github.com/Real-Dev-Squad/tiny-site-backend/blob/develop/config/core-config.go
    2. https://github.com/Real-Dev-Squad/wisee-backend/blob/develop/src/config/core-config.go

Reply:
https://github.com/Real-Dev-Squad/feature-flag-backend/issues/134

Will be addressed in this ticket.